client/src/components/EmailIcon.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import EIcon from "../assets/icons/iconfinder_icon-email_211660.svg";
+import styled from "styled-components";
+
+const EmailIcon = () => {
+  return (
+    <>
+      <Container>
+        <ELink href='mailto:someone@email.com' target="blank">
+        <Circle>
```

Comment:
If you really want to create a separate component for all these icons then this one could take a prop of `email` that you can pass in from `OverviewProfileCard` and `ViewMyProfile` like you did with other icons.